docs/en/operations/settings/merge-tree-settings.md

```diff
@@ -1133,10 +1133,24 @@ By using `ORDER BY time DESC` in the query, `ReadInOrder` is applied.
 
 ## cache_populated_by_fetch
 
-A Cloud only setting. 
+:::note
+This setting applies only to ClickHouse Cloud.
+:::
 
 When `cache_populated_by_fetch` is disabled (the default setting), new data parts are loaded into the cache only when a query is run that requires those parts.
 
-If enabled, `cache_populated_by_fetch` will instead cause all nodes to load new data parts from storage into their cache without requiring a query to trigger such an action. 
+If enabled, `cache_populated_by_fetch` will instead cause all nodes to load new data parts from storage into their cache without requiring a query to trigger such an action.
 
 Default value: 0.
+
+## enable_minmax_index_for_all_numeric_columns
```

Comment:
Setting names `add_minmax_index_for_numeric_columns` and `add_minmax_index_for_string_columns` would be a little bit more descriptive and less redundant (no `all`). Like to rename these settings?

Reply:
renamed settings